Extension/src/LanguageServer/Providers/workspaceSymbolProvider.ts

```diff
@@ -14,8 +15,13 @@ export class WorkspaceSymbolProvider implements vscode.WorkspaceSymbolProvider {
     }
 
     public async provideWorkspaceSymbols(query: string, token: vscode.CancellationToken): Promise<vscode.SymbolInformation[]> {
+        if (!query) {
+            return [];
+        }
+
         const params: WorkspaceSymbolParams = {
-            query: query
+            query: query,
+            experimentEnabled: await isExperimentEnabled('CppTools1')
```

Comment:
Just checking, is this the correct capitalization? Experiments are case-sensitive and I assumed we would call them 'cpptools1', 'cpptools2', etc.

Reply:
Yes, the casing is correct (Alex was *very* specific 😄 )